kedro/framework/cli/utils.py

```diff
@@ -294,6 +294,26 @@ def split_string(ctx, param, value):  # pylint: disable=unused-argument
     return [item.strip() for item in value.split(",") if item.strip()]
 
 
+# pylint: disable=unused-argument,missing-param-doc,missing-type-doc
+def split_node_names(ctx, param, to_split: str) -> List[str]:
+    """Split string by comma, ignoring commas enclosed by an even number
+    of \" and \' characters.
+
+    Args:
+        to_split: the string to split safely
+
+    Returns:
+        A list containing the result of safe-splitting the string.
+    """
+    # Only split on commas with an even number of ' and " ahead
```

Comment:
In the PR description you said: "Commas surrounded by an even number of " and ' characters will no longer split the arguments to kedro run options", and that's also what the regex does, so shouldn't this comment read "Don't split on commas with an even number of ' and " ahead"?